pkg/util/test/util.go

```diff
@@ -288,6 +288,42 @@ func ValidateStorageCluster(
 
 	pxEndpoint = fmt.Sprintf("%s:%d", svc.Spec.ClusterIP, servicePort)
 
+	if err := validatePodsByName(cluster.Namespace, "portworx-pvc-controller", timeout, interval); err != nil {
```

Comment:
PVC controller is not always deployed. You can refer the following function to see when pvc controller is enabled - https://github.com/libopenstorage/operator/blob/master/drivers/storage/portworx/component/pvccontroller.go#L58

I would recommend duplicating a similar function in test.

Reply:
Done